src/app/utils/Log.ts

```diff
@@ -1,150 +1,106 @@
+			logger.info(statement);
 			Sentry.addBreadcrumb({
 				level: Sentry.Severity.Info,
 				message: statement,
 			});
 		}
 	},
 
-	warn(statement: string | any, options?: Omit<LogOptions, 'level'>): void {
+	warn(statement: string | any): void {
 		if (process.env.NODE_ENV != 'production' || !logger.warn) {
 			// eslint-disable-next-line no-console
 			console.log(statement);
 		} else {
-			logger.warn(statement, options);
+			logger.warn(statement);
 			Sentry.addBreadcrumb({
 				level: Sentry.Severity.Warning,
 				message: statement,
 			});
 		}
 	},
 
-	debug(statement: string | any, options?: Omit<LogOptions, 'level'>): void {
+	debug(statement: string | any): void {
 		if (process.env.NODE_ENV != 'production' || !logger.debug) {
 			// eslint-disable-next-line no-console
 			console.debug(statement);
 		} else {
-			logger.debug(statement, options);
+			logger.debug(statement);
 		}
 	},
 
-	error(statement: string | any, options?: Omit<LogOptions, 'level'>): void {
+	error(statement: string | any): void {
 		if (process.env.NODE_ENV != 'production' || !logger.error) {
 			// eslint-disable-next-line no-console
 			console.error(statement);
 		} else {
-			logger.error(statement, options);
+			logger.error(statement);
 			Sentry.addBreadcrumb({
 				level: Sentry.Severity.Error,
 				message: statement,
 			});
 		}
 	},
 
-	fatal(statement: string | any, options?: Omit<LogOptions, 'level'>): void {
+	fatal(statement: string | any): void {
 		if (process.env.NODE_ENV != 'production' || !logger.fatal) {
 			// eslint-disable-next-line no-console
 			console.error(statement);
 		} else {
-			logger.fatal(statement, options);
+			logger.fatal(statement);
 			Sentry.addBreadcrumb({
 				level: Sentry.Severity.Fatal,
 				message: statement,
 			});
 		}
 	},
 
-	trace(statement: string | any, options?: Omit<LogOptions, 'level'>): void {
+	trace(statement: string | any): void {
 		if (process.env.NODE_ENV != 'production' || !logger.trace) {
 			// eslint-disable-next-line no-console
 			console.log(statement);
 		} else {
-			logger.trace(statement, options);
+			logger.trace(statement);
 		}
 	},
 
-	log(statement: string | any, options?: Omit<LogOptions, 'level'>): void {
-		if (process.env.NODE_ENV != 'production') {
-			// eslint-disable-next-line no-console
-			console.log(statement);
-		}
-		logger.log(statement, options);
-		Sentry.addBreadcrumb({
-			level: Sentry.Severity.Log,
-			message: statement,
-		});
-	},
-
-	// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
-	addMetaProperty(key: string, value: any): void {
-		logger.addMetaProperty(key, value);
-	},
-
-	removeMetaProperty(key: string): void {
-		logger.removeMetaProperty(key);
-	},
-
 	flush(): void {
 		logger.flush();
 	},
 };
 
 export const LogUtils = {
 	logCommandStart(ctx: CommandContext): void {
-		Log.info(`/${ctx.commandName} ran ${ctx.user.username}#${ctx.user.discriminator}`, {
-			indexMeta: true,
-			meta: {
```

Comment:
Same as above for RateLimit.ts and Checkout.ts. Since we're in the process of updating some of the log captures, might as well solve this now.